OSDOCS-4632: Observing network traffic (#54085)
openshift/openshift-docs: modules/network-observability-working-with-topology.adoc

// Module included in the following assemblies:
//
// network_observability/observing-network-traffic.adoc
:_content-type: PROCEDURE
[id="network-observability-working-with-topology_{context}"]

> = Working with Topology

Reviewer: ```suggestion
= Working with the Topology view```

> As an administrator, you can navigate to topology view by executing the following:

Reviewer: ```suggestion
As an administrator, you can navigate to the *Topology* view to see the details and metrics of the component.
```

. On the left navigation panel, click *Observe* → *Network Traffic*.
. In the *Network Traffic* page, click the *Topology* tab.

> You can click each component in the topology to view the details and metrics of the component.

Reviewer: ```suggestion
You can click each component in the *Topology* to view the details and metrics of the component.
```
